components/raftstore-v2/src/worker/tablet.rs

```diff
@@ -233,12 +235,53 @@ impl<EK> Task<EK> {
     }
 }
 
+struct TrimJob<EK> {
+    tablet: EK,
+    start: Box<[u8]>,
+    end: Box<[u8]>,
+    cb: Box<dyn FnOnce() + Send>,
+    start_time: Instant,
+    slow_threshold: Duration,
+    // Tablet's latest seqno when this job is created. We should wait for the release of all
+    // snapshots before this fence.
+    // In reality, all dirty tablet is created from checkpoint. There shouldn't be any stale
+    // snapshot.
+    seqno_fence: u64,
+}
+
+impl<EK: KvEngine> TrimJob<EK> {
+    fn check(&mut self, logger: &Logger) -> bool {
+        let res = self
+            .tablet
+            .get_oldest_snapshot_sequence_number()
+            .unwrap_or(u64::MAX)
+            > self.seqno_fence;
```

Comment:
Does `oldest_snapshot_sequence` change when there is no write?

Reply:
Good catch.

Reply:
@tabokie Is there any action needed for this comment? I don't find any code update on this

Reply:
The reverted commit removes this part.